tests/integration2/pilot/sidecar_api_test.go

```diff
@@ -194,5 +194,5 @@ func validateListenersNoConfig(t *testing.T, response *structpath.Structpath, mi
 // - Do cleanup before exit
 // - process testing specific flags
 func TestMain(m *testing.M) {
-	framework.Run("sidecar_api_test", m)
+	_, _ = framework.Run("sidecar_api_test", m)
```

Comment:
This is broken. This will cause the tests to always pass in the lab.

Reply:
@ozevren are you saying that my change broke this? Or that it was already broken?

Reply:
@ozevren took a look at other examples ... I think I have the fix now. PTAL

Reply:
It was broken. During the last refactoring, the return values were silently ignored as framework.Run was executing the test but not returning the right error code during exit. This caused lab to have false positives. Andy fixed them circa December, I believe.

In the newly refactored one, I reverted things to the model of framework.Run() directly exiting, so that it is impossible to make this mistake.